src/AlgoliaSearch.js

```diff
@@ -118,54 +119,56 @@ AlgoliaSearch.prototype = {
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  deleteIndex: function(indexName, callback) {
-    return this._jsonRequest({ method: 'DELETE',
-              url: '/1/indexes/' + encodeURIComponent(indexName),
-              hostType: 'write',
-              callback: callback });
+  deleteIndex: function deleteIndex(indexName, callback) {
```

Comment:
I was not aware that some browsers would do such thing :) And it's already a warning, good for me (I haven't seen it in the helper though).